### PR TITLE
Add targets option to @babel/plugin-transform-runtime

### PR DESCRIPTION
**Pull request by TomerAberbach**
Adds documentation for a new `@babel/plugin-transform-runtime` `targets` option.

### CODE REVIEW

**Comment by dilyanpalauzov** [comment]
I am glad you got the point.

I propose first to find the wordings on the same paragraphs at https://github.com/babel/website/issues/2245 and https://github.com/babel/website/pull/2253 and then copy-paste here the result.

**Comment by TomerAberbach** [comment]
> I am glad you got the point.
> 
> I propose first to find the wordings on the same paragraphs at #2245 and #2253 and then copy-paste here the result.

Let's wait until #2252 is merged before copy-pasting its changes here (if I copy from #2252 now, then I might miss out on changes made to the PR after I copy, but before it's merged). I'll make sure this PR doesn't get merged until #2252 is merged.

**Comment by TomerAberbach** [comment]
@dilyanpalauzov Just letting you know it was decided that babel/babel#11572 won't support `configPath` and `ignoreBrowserslistConfig` initially so I've changed the wording now.

File: docs/plugin-transform-runtime.md [hunk 1]
```diff
@@ -123,8 +124,93 @@ For more information, see [Helper aliasing](#helper-aliasing).
...
+
+Example environments: `chrome`, `opera`, `edge`, `firefox`, `safari`, `ie`, `ios`, `android`, `node`, `electron`.
+
+Sidenote, if the `targets` option is not set and `corejs` is not set to `false`, then `@babel/plugin-transform-runtime` will transform all ECMAScript 2015+ code.
+
+> We don't recommend using `plugin-transfrom-runtime` this way because it doesn't take advantage of its ability to target specific browsers.
+
+```json
+{
+  "presets": ["@babel/plugin-transform-runtime"]
```
[the remark this answers is not in the document]
Comment (TomerAberbach) [reply docs/plugin-transform-runtime.md:164]: Good catch! Thanks

**Pull request closed without merging**